index.html

```diff
@@ -11,6 +11,7 @@
       <base href="/">
       <!-- @if ENV='dev' -->
       <link rel="stylesheet" href="dist/give.css" type="text/css">
+      <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.css">
```

Comment:
Ya that question mark has been missing for a while. AEM seems to include font-awesome. I guess this is fine for development.